firefox_puppeteer/ui/toolbars.py

```diff
@@ -6,13 +6,21 @@
 
 from ..api.keys import Keys
 from ..api.l10n import L10n
-from ..base import BaseLib
+from ..base import UIBaseLib
 from ..decorators import use_class_as_property
 
 
-class NavBar(BaseLib):
+class NavBar(UIBaseLib):
     """Provides access to the DOM elements contained in the
     navigation bar as well as the location bar."""
+    def __init__(self, marionette_getter, window):
+        UIBaseLib.__init__(self, marionette_getter, window)
+        # TODO: A "utility" module that sets up the client directly would be
+        # useful here.
+        self._locationbar = None
+
+        self.l10n = L10n(self.get_marionette)
+        self.keys = Keys(self.get_marionette)
```

Comment:
I don't see that we would need those two classes in the NavBar class. I think they can be removed. Or do I miss something?